CHANGELOG.md

```diff
@@ -23,12 +23,14 @@ The types of changes are:
 * Subject Request Details page [#563](https://github.com/ethyca/fidesops/pull/563)
 * Restart Graph from Failure [#578](https://github.com/ethyca/fidesops/pull/578)
 * Redis SSL Support [#611](https://github.com/ethyca/fidesops/pull/611)
+* Celery as a dependency for use in the execution layer [#610](https://github.com/ethyca/fidesops/pull/610)
 * Cache and Surface Resume/Restart Instructions [#591](https://github.com/ethyca/fidesops/pull/591)
```

Comment:
This changelog is looking out of date?

Reply:
Anything specific you can see that's missing here? Merging in `main` to this branch isn't yielding any conflicts or extra lines.

Reply:
I just thought 1.5.3 had already been released -

Reply:
Chatted in person, my mistake, 1.5.3 was released from a different branch